causalml/inference/nn/utils.py

```diff
@@ -0,0 +1,97 @@
+import tensorflow as tf
+import keras.backend as K
+from keras.engine.topology import Layer
+from keras.metrics import binary_accuracy
+
+
+def binary_classification_loss(concat_true, concat_pred):
```

Comment:
Could you add docstrings to functions here?

Reply:
docstrings added in latest push